xml/System.Reflection/Assembly.xml

```diff
@@ -5235,20 +5237,21 @@ The ability to execute code in remote assemblies is disabled. See <see href="htt
         <summary>Gets the full path or UNC location of the loaded file that contains the manifest.</summary>
         <value>The location of the loaded file that contains the manifest. If the loaded file was shadow-copied, the location is that of the file after being shadow-copied. If the assembly is loaded from a byte array, such as when using the <see cref="M:System.Reflection.Assembly.Load(System.Byte[])" /> method overload, the value returned is an empty string ("").</value>
         <remarks>
-          <format type="text/markdown"><![CDATA[  
-  
-## Remarks  
- To get the location before the file has been shadow-copied, use the <xref:System.Reflection.Assembly.CodeBase%2A> property.  
-  
-   
-  
-## Examples  
- The following example displays the location of the loaded file that contains the manifest.  
-  
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+To get the location before the file has been shadow-copied, use the <xref:System.Reflection.Assembly.CodeBase%2A> property.
```

Comment:
Is it worth noting that `CodeBase` is obsolete now? I don't think shadow copying is supported in .NET Core, so this line probably only applies to the desktop FX.